packages/ra-ui-materialui/src/list/filter/FilterButton.tsx

```diff
@@ -91,12 +91,13 @@ const FilterButton = (props: FilterButtonProps): JSX.Element => {
                 anchorEl={anchorEl.current}
                 onClose={handleRequestClose}
             >
-                {hiddenFilters.map((filterElement: JSX.Element) => (
+                {hiddenFilters.map((filterElement: JSX.Element, index) => (
                     <FilterButtonMenuItem
                         key={filterElement.props.source}
                         filter={filterElement}
                         resource={resource}
                         onShow={handleShow}
+                        autoFocus={index === 0}
```

Comment:
I don't get why this is necessary to fix the keyboard navigation. Can you explain?

Reply:
I couldn't get it to work otherwise, I'm forcing the focus on the first menu item, do you have a suggestion with aria attributes?

Reply:
No but I fail to find a difference between our code and the MUI v4 demos for the menu

Reply:
> No but I fail to find a difference between our code and the MUI v4 demos for the menu

What I've noticed is that they don't have a forwardRef `MenuItem` inside a `Menu` in their examples.
It works out of the box without this, but of course, we need it.

Reply:
For example, this works:

```jsx
<Menu
    open={open}
    anchorEl={anchorEl.current}
    onClose={handleRequestClose}
>
    {hiddenFilters.map((filterElement: JSX.Element, index) => (
        <MenuItem
            key={filterElement.props.source}
            //filter={filterElement}
            //resource={resource}
            //onShow={handleShow}
            //autoFocus={index === 0}
        >
            <FieldTitle
                label={filterElement.props.label}
                source={filterElement.props.source}
                resource={resource}
            />
        </MenuItem>
    ))}
</Menu>

Reply:
Of course it would be a BC